toolchains/Toolchain-QNX7-QCC-AArch64.cmake

```diff
@@ -0,0 +1,6 @@
+set(CMAKE_SYSTEM_PROCESSOR aarch64)
```

Comment:
@BSolonenko this toolchain is redundant

Reply:
@AKalinich-Luxoft Fixed in b2b51c312fc57007af9a6626c0e88bfe2f0b594e